packages/components/filter-bar/pipes/pipe-date.ts

```diff
@@ -0,0 +1,206 @@
+import { ChangeDetectionStrategy, Component, inject, viewChild, ViewChild, ViewEncapsulation } from '@angular/core';
+import { FormControl, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ENTER } from '@koobiq/cdk/keycodes';
+import { KbqButton, KbqButtonModule, KbqButtonStyles } from '@koobiq/components/button';
+import {
+    DateAdapter,
+    DateFormatter,
+    KBQ_VALIDATION,
+    KbqComponentColors,
+    KbqFormattersModule,
+    PopUpPlacements
+} from '@koobiq/components/core';
+import { KbqDatepickerModule } from '@koobiq/components/datepicker';
+import { KbqDividerModule } from '@koobiq/components/divider';
+import { KbqFormFieldModule } from '@koobiq/components/form-field';
+import { KbqIcon } from '@koobiq/components/icon';
+import { KbqInputModule } from '@koobiq/components/input';
+import { KbqListModule, KbqListSelection } from '@koobiq/components/list';
+import { KbqPopoverModule, KbqPopoverTrigger } from '@koobiq/components/popover';
+import { KbqTimepickerModule } from '@koobiq/components/timepicker';
+import { KbqTitleModule } from '@koobiq/components/title';
+import { KbqDateTimeValue } from '../filter-bar.types';
+import { KbqBasePipe } from './base-pipe';
+import { KbqPipeButton } from './pipe-button';
+import { KbqPipeState } from './pipe-state';
+import { KbqPipeTitleDirective } from './pipe-title';
+
+@Component({
+    standalone: true,
+    selector: 'kbq-pipe-date',
+    templateUrl: 'pipe-date.html',
+    styleUrls: ['base-pipe.scss', 'pipe-date.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+    providers: [
+        {
+            provide: KBQ_VALIDATION,
+            useValue: { useValidation: false }
+        },
+        {
+            provide: KbqBasePipe,
+            useExisting: this
```

Comment:
кажется что вместо `this` нужно явно указать
```suggestion
            useExisting: KbqPipeDateComponent
```